test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,25 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // Contains at least one null value
+        assertContainsNull((Object) null);
+        assertContainsNull(null, 1, "hello");
+        assertContainsNull(1, null, "bye", null);
```

Comment:
Sometimes when adding test cases, think about how the method treat this new test case differently from the other existing ones.